spec/services/proofing/lexis_nexis/ddp/verification_request_spec.rb

```diff
@@ -81,6 +81,25 @@
         expected_json = JSON.parse(LexisNexisFixtures.ddp_authentication_request_json)
         expect(response_json).to eq(expected_json)
       end
+
+      context 'with service provider associated with user' do
+        let(:applicant) do
+          {
+            threatmetrix_session_id: 'UNIQUE_SESSION_ID',
+            email: 'test@example.com',
+            request_ip: '127.0.0.1',
+            uuid_prefix: 'SPNUM',
```

Comment:
that Applicant is for IDV verification not account creation, that should be a different applicant. Can move it around to be clearer though.

Reply:
On that note, rather than calling `merge` to `base_json`, should the `base_json` (`account_creation_request.json`) be updated to reflect the `local_attrib_1` we're expecting to be included, in the same way it's baked into IdV's `request.json` ?

https://github.com/18F/identity-idp/blob/f94b900fd4032eb030d7346b4fca8b80188e4674/spec/fixtures/proofing/lexis_nexis/ddp/request.json#L25

Reply:
I would say We can make one request that defaults it and another that does not. We do not expect users to have a service provider on account creation so I wanted to ensure we have the usecase for both users with local_attib_1 set up and one with it blank. We can make a second request json for both use cases.

Reply:
That's fair 👍